browser.py

```diff
@@ -243,9 +243,23 @@ def __page_removed_cb(self, notebook, child, pagenum):
 
     def __new_tab_cb(self, browser, url):
         new_browser = self.add_tab(next_to_current=True)
+        self.emit('focus-url-entry')
         new_browser.load_uri(url)
         new_browser.grab_focus()
 
+    def __new_tab_background_cb(self, browser, url):
+        browser = Browser(self._activity)
```

Comment:
To be consistent with `__new_tab_cb` and not confuse with method parameter of the same name, local variable `browser` could be `new_browser`.